playground/nats/Nats.AppHost/aspire-manifest.json

```diff
@@ -3,9 +3,13 @@
   "resources": {
     "nats": {
       "type": "container.v0",
-      "connectionString": "nats://{nats.bindings.tcp.host}:{nats.bindings.tcp.port}",
+      "connectionString": "nats://nats:{nats-password.value}@{nats.bindings.tcp.host}:{nats.bindings.tcp.port}",
```

Comment:
what's the point of adding user:pass in the url? NATS .NET client doesn't support this notation so it doesn't have a functional purpose.